mak/Makefile.download

```diff
@@ -183,11 +183,14 @@ ifdef expat_pkg
 	  if test ! -d pkgs; then mkdir pkgs; fi
 	  cd pkgs && \
 	  wget -nv $(expat_srcpath) -O $(expat_pkg) && \
+	  wget -nv $(expat_srcpath).asc -O $(expat_pkg).asc && \
 	  cd .. && \
-	  tar -xzf pkgs/$(expat_pkg)
+	  tar -xjf pkgs/$(expat_pkg)
 else
```

Comment:
Can you get away with just `tar -xf ...`?  In other words assume GNU tar or similar.

Reply:
See comment below under Makefile.gather gather-expat-release, this shouldn't be assumed or inferred IMO.